cmd/utils/dev.go

```diff
@@ -367,6 +367,9 @@ func AskForOptions(options []string, label string) (string, error) {
 
 //AskIfOktetoInit asks if okteto init should be executed
 func AskIfOktetoInit(devPath string) bool {
+	if devPath == "" {
+		devPath = DefaultManifest
```

Comment:
Is that the name of the one we'll create if the user selects "y".

Reply:
Yes, When we call `okteto init` without adding the file it will create it with the name of `DefaultManifest`